lib/src/main/java/org/team100/lib/indicator/LEDIndicator.java

```diff
@@ -62,21 +63,41 @@ private State(Color color) {
     private final AddressableLEDBuffer buffer;
     private final AddressableLEDBuffer blackBuffer;
     private final List<LEDStrip> leds = new ArrayList<>();
-    private boolean flashing = false;
+    private boolean flashing;
+    private final int m_numstrips;
+    private final int m_numLEDs;
 
-    public LEDIndicator(int port, LEDStrip... strips) {
+    public LEDIndicator(int port, int numLEDs, LEDStrip... strips) {
```

Comment:
the purpose of the "numLEDs" parameter is not clear to me.  what does this mean?